hathor/manager.py

```diff
@@ -1223,6 +1225,14 @@ def check_sync_state(self):
 
             self.lc_check_sync_state.stop()
 
+    def set_cmd_path(self, path: str) -> None:
+        """Set the cmd path, where sysadmins can place files to communicate with the full node."""
```

Comment:
I think we will need to update the hathor-core SOP with this information.

Reply:
@luislhl You are right. And it will change considerably more after https://github.com/HathorNetwork/hathor-core/pull/587.